install.sh

```
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Function to read environment variables from .env file
+load_env() {
+    if [ -f .env ]; then
+        export $(cat .env | grep -v '#' | awk '/=/ {print $1}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)